modules/nf-core/odgi/layout/main.nf

```diff
@@ -0,0 +1,35 @@
+process ODGI_LAYOUT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::odgi=0.8.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/odgi:0.8.0--py310hc8f18ef_0':
+        'quay.io/biocontainers/odgi:0.8.0--py310hc8f18ef_0' }"
+
+    input:
+    tuple val(meta), path(graph)
+
+    output:
+    tuple val(meta), path("*.lay"), optional: true, emit: lay
+    tuple val(meta), path("*.tsv"), optional: true, emit: tsv
+    path "versions.yml"                           , emit: versions
```

Comment:
Since either lay, tsv, or both outputs are possible, we have to keep these channels optional.

Reply:
Optional output means that the specified files aren't always part of the process. Since the tool will now create both files every time, it will always be there and isn't optional anymore

Reply:
Yes, but I want these to stay optional :)